frontend/app/routes/$lang/_protected/access-to-governmental-benefits/view.tsx

```diff
@@ -160,8 +160,7 @@ export default function AccessToGovernmentalsBenefitsView() {
           <InlineLink
             routeId="$lang/_protected/access-to-governmental-benefits/edit"
             params={{ ...params }}
-            className="external-link"
-            target="_blank"
+            className="font-lato font-semibold"
```

Comment:
I noticed that these class names are used elsewhere on links in the `/access-to-governmental-benefits` pages, but we don't need them. The inline links should have the same font style as the body text.

Reply:
Okay, removed in commit 555fe7b0b8f43fd11acdea439e55bc24edbf83d0.